src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.IntakeConstants;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeCommand extends Command {
+    IntakeSubsystem m_Intake;
+
+    /** Creates a new IntakeCommand. */
+    public IntakeCommand(IntakeSubsystem intake) {
+        m_Intake = intake;
+        addRequirements(m_Intake);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        if (!isFinished()) {
```

Comment:
I don't understand this logic. execute should never run if isFinished is true. Also isFinished always returns false. Just a bit confusing.

Reply:
Oh this is strange. Do we know who wrote that?